apps/server/src/modules/files-storage/client/s3-client.adapter.spec.ts

```diff
@@ -46,131 +46,193 @@ describe('S3ClientAdapter', () => {
 
 		service = module.get(S3ClientAdapter);
 		client = module.get('S3_Client');
-		client.config.endpoint = jest.fn().mockResolvedValue({ protocol: '' });
+	});
+
+	const clientSendMockResolved = () => {
```

Comment:
This method is weirdly positioned and also only used once. Why not put the code in the setup where it is used?